spec/factory_girl/registry_spec.rb

```diff
@@ -57,4 +57,19 @@
     subject.clear
     expect(subject.count).to be_zero
   end
+
+  it "removes a factory" do
+    subject.register(:object_name, registered_object)
+    expect do
+      expect(subject.remove(:object_name)).to eq(registered_object)
```

Comment:
That makes sense. In that case, if the return value is provided but not significant to us as a library, it might make sense not to test the result, but assert on the behavior.

If we do in fact want to support that piece of API (specifically, if you delete a factory, you'll get the deleted factory as the return value), then leaving the test in makes sense. I don't feel strongly, was more just trying to understand if the return value is important.